man/systemd-journal-gatewayd.service.xml

```diff
@@ -198,6 +198,15 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><uri>/fields</uri></term>
+
+        <listitem><para>Return the list of fields present in the logs.</para>
+        </listitem>
+
+        <xi:include href="version-info.xml" xpointer="v255"/>
```

Comment:
Please insert version info before `</listitem>`, otherwise CIs are unhappy.

Reply:
This code is no longer in this PR, but I've fixed this point in #29484 .